ibm_catalog.json

```diff
@@ -92,7 +93,58 @@
                 }
               ]
             }
-          }
+          },
+          {
+            "label": "Agents",
+            "name": "agents",
+            "install_type": "extension",
+            "working_directory": "solutions/agents",
+            "dependencies": [
+             {
+                "flavors": [
+                 "standard"
+                ],
+                "id": "95fccffc-ae3b-42df-b6d9-80be5914d852",
+                "name": "deploy-arch-ibm-slz-ocp",
+                "version": ">=v5.20.0"
```

Comment:
why are you requiring v5.20.0 as the minimum version? The standard flavour has existed since v3.0.0

![image](https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/assets/29608224/b6dc658f-b09b-4835-b745-cf41a8bfbb34)

The quickstart flavor came in v5.20.0, so maybe add a separate entry for that?

Reply:
@ocofaigh did we not suggest to use the latest one?
I think...if we do so, the existing user may not be able to use this DA. Fine, will change this and add a separate entry for quickstart.

Also, please check this thread - https://github.com/terraform-ibm-modules/terraform-ibm-observability-da/pull/4#discussion_r1538921764
we are yet to decide on the region input. If crn seems to be better option, I'll make change for both. Else need to leave as-is.

Reply:
@maheshwarishikha I already replied to that thread - did you not see it?

Reply:
Now I see. Thanks for the quick discussion too.